sdk/python/kfp/v2/dsl/serialization_utils.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Utilities for serialization/deserialization."""
+from typing import Any, Dict
+import yaml
+
+
+# Serialize None as blank instead of 'null'.
+def _represent_none(self, _):
+  return self.represent_scalar('tag:yaml.org,2002:null', '')
+
+
+class NoneAsBlankDumper(yaml.SafeDumper):
```

Comment:
nit: Seems to be internal only, so maybe `_NoneAsBlankDumper`?

Reply:
Done.